app/components/post/post.js

```diff
@@ -114,7 +114,23 @@ export default class Post extends Component {
 
     viewUserProfile = () => {
         this.props.actions.goToUserProfile(this.props.user.id);
-    }
+    };
+
+    renderMessage = (style, messageStyle) => {
+        return (
+            <TouchableHighlight onPress={this.handlePress}>
+                <View style={{flex: 1}}>
+                    {this.renderReplyBar(style)}
```

Comment:
This looks like it's going to render two reply bars for comments that aren't the start of the message chain since `this.renderReplyBar` is still called in `render` in some cases